pkg/controller/jobs/pod/pod_controller_test.go

```diff
@@ -3371,6 +3372,138 @@ func TestReconciler(t *testing.T) {
 				},
 			},
 		},
+		"workload is created with correct labels for a single pod": {
+			pods: []corev1.Pod{*basePodWrapper.
+				Clone().
+				Label("kueue.x-k8s.io/managed", "true").
+				Label("toCopyKey", "toCopyValue").
+				Label("dontCopyKey", "dontCopyValue").
+				KueueFinalizer().
+				KueueSchedulingGate().
+				Queue("test-queue").
+				Obj()},
+			wantPods: nil,
+			reconcilerOptions: []jobframework.Option{
+				jobframework.WithLabelKeysToCopy([]string{"toCopyKey"}),
```

Comment:
nit: add also a label key name that is absent in the pod.

Reply:
Done